subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/ConfigurationCacheState.kt

```diff
@@ -84,14 +84,15 @@ import java.io.OutputStream
 
 
 internal
-enum class StateType {
-    Work, Model, Entry, BuildFingerprint, ProjectFingerprint, IntermediateModels, ProjectMetadata
+enum class StateType(val encrypted: Boolean = false) {
```

Comment:
s/encryptedf/encryptable/

Also, have yet to test which set of state files should really be encryptable